recipes/tqdm-multiprocess/meta.yaml

```
@@ -0,0 +1,44 @@
+{% set name = "tqdm-multiprocess" %}
+{% set version = "0.0.11" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  - url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/tqdm-multiprocess-{{ version }}.tar.gz
+    sha256: a74002a1222ea9cbe8cdc9bd460108c6009be359621fbee9b92d0515d4d180f7
+  - url: https://raw.githubusercontent.com/EleutherAI/tqdm-multiprocess/master/requirements-dev.txt
```

This should also be conditional on the version.
```suggestion
  - url: https://raw.githubusercontent.com/EleutherAI/tqdm-multiprocess/{{ version }}/requirements-dev.txt
```

---

@xhochy I do not disagree, however this package has no official releases or tags in GitHub.  https://github.com/EleutherAI/tqdm-multiprocess/tags

---

😭